templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>You want delete {{ manufacturer }}</h1>
```

Comment:
There is a grammatical error in the heading. It should be 'Do you want to delete {{ manufacturer }}?' to make it grammatically correct.